.tekton/pull-request.yaml

```diff
@@ -81,7 +81,6 @@ spec:
               image: quay.io/redhat-appstudio/pull-request-builds:build-definitions-utils-{{revision}}
               # per https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
```

Comment:
Maybe this is also out of place? Not sure if there's any sense in copying this everywhere.